docs/flow/modin/engines/base/io.rst

```diff
@@ -0,0 +1,95 @@
+:orphan:
+
+IO Module Description
+"""""""""""""""""""""
+
+High-Level Data Import Operation Workflow
+'''''''''''''''''''''''''''''''''''''''''
+
+**Note**: ``read_csv`` on pandas backend and Ray engine was taken as an example
+in this chapter for reader convenience. For other import functions workflow and
+classes/functions naming convension will be the same.
+
+After user calls high-level ``modin.pandas.read_csv`` function, call is forwarded
```

Comment:
A diagram would really help here, it is hard to follow the logic.

Reply:
Ok, diagram was added.